src/ui/pages/equipment/asset/search/AssetSearch.js

```diff
@@ -19,6 +20,12 @@ export default class AssetSearch extends Component {
     }
 
     render() {
+        const filters = GridTools.parseGridFilters(GridTools.getURLParameterByName('gridFilters'));
+        const setSearchFilters = (gridFilters) => {
+            const params = GridTools.replaceUrlParam('gridFilters', GridTools.stringifyGridFilters(gridFilters));
+            this.props.history.push(params);
+        }
```

Comment:
This will be migrated to the new grid component, and that's why it is like this now. In any case, the proper way would be too have this feature embedded in the eam-components anyway, as it is still repeated code with your solution.